src/js/components/Form/stories/ControlledInput.js

```diff
@@ -104,6 +107,19 @@ export const ControlledInput = () => {
                 onChange={event => setAge(event.target.value)}
               />
             </FormField>
+            <FormField
+              htmlFor="file"
+              required
+              label="FileInput"
+              name="file"
+              pad
+            >
+              <FileInput
+                name="file"
+                id="file"
+                onChange={event => setFile(event.target.files)}
```

Comment:
Seeing console error, i think relate to my suggestion above:
<img width="466" alt="Screen Shot 2021-05-17 at 10 59 20 AM" src="https://user-images.githubusercontent.com/12522275/118535264-19298800-b6ff-11eb-8c8a-469b666b9588.png">